spec/aep-64/README.md

```diff
@@ -0,0 +1,76 @@
+---
+aep: 64
+title: "JWT authentication for provider API"
+author: Artur Troian (@troian)
+status: Draft
+type: Standard
+category: Core
+created: 2025-04-03
+updated: 2025-04-03
+roadmap: major
+---
+
+## Motivation
+
+Enhance tenant authentication to the provider gateway using JWT allows client API to retain communication with leases during blockchain maintenance.
+
+## Summary
+
+mTLS authentication is great security mechanism, however, with current implementation when blockchain acts as root of trust, it becomes complicated for API clients to retain communication with leases during blockchain maintenance windows. There are also other limitations like granual access. Certificate gives access to all all leases and all features, tho granual access is possible via certificate, implementing and maintaining it for clients is challenging topic. JWT on other hand is wide adopted approach for authentication and authorization.
+
+## Technical details
+
+Akash uses asymetric keys (ECDSA) for wallets with secp256k1 curve and allows for signing and encrypting messages.
+For this AEP, we will be focused on signing part only.
```

Comment:
Off-topic (for this AEP): Through mTLS, the SDLs are encrypted using the public key of the provider which guarantees that only the provider is able to decrypt the SDL (IIRC). How will we prevent this with client-side JWT? Will we still encrypt SDL using the providers public key? What's expected from the context of this AEP?

Reply:
Manifest is not encrypted with the provider's public key, it is rather being send via TLS transport